packages/components/psammead-navigation/README.md

```diff
@@ -39,52 +44,78 @@ The `@bbc/psammead-navigation` package is a set of two components, `NavigationUl
 
 ## Usage
 
 ```jsx
 import React from 'react';
+import { CanonicalSwipeableNavigation } from '@bbc/psammead-navigation/swipeable-navigation';
 import Navigation, {
   NavigationUl,
   NavigationLi,
 } from '@bbc/psammead-navigation';
 import { latin } from '@bbc/gel-foundations/scripts';
 
 <Navigation>
-  <NavigationUl>
-    <NavigationLi
-      url="/"
-      script={latin}
-      active="true"
-      currentPageText="Current Page"
-      service="news"
-    >
-      Home
-    </NavigationLi>
-    <NavigationLi url="/sport" script={latin} service="news">
-      {Sport}
-    </NavigationLi>
-    <NavigationLi url="/weather" script={latin} service="news">
-      {Weather}
-    </NavigationLi>
-  </NavigationUl>
+  <CanonicalSwipeableNavigation>
+    <NavigationUl>
+      <NavigationLi
+        url="/"
+        script={latin}
+        active="true"
+        currentPageText="Current Page"
+        service="news"
+      >
+        Home
+      </NavigationLi>
+      <NavigationLi url="/sport" script={latin} service="news">
+        {Sport}
+      </NavigationLi>
+      <NavigationLi url="/weather" script={latin} service="news">
+        {Weather}
+      </NavigationLi>
+    </NavigationUl>
+  </CanonicalSwipeableNavigation>
 </Navigation>;
 ```
 
 ### When to use this component
 
 The `Navigation` is designed to show a navigation bar on `index` pages, which will show all sections on a site. If there are too many items to fit on one line, the items will wrap to the next lines.
 
+On the other hand, with `CanonicalSwipeableNavigation` or `AmpSwipeableNavigation` we can make the list to remain on one line and to be horizontally swipeable to allow access to further links, under 600px.
+
 ### Accessibility notes
 
 The Navigation has a [`navigation` landmark](https://www.w3.org/TR/wai-aria-practices/examples/landmarks/navigation.html) to provide a way to identify links that are intended to be used for navigation.
 
-It includes a ["skip link"](https://www.w3.org/TR/WCAG20-TECHS/G1.html) giving users the option to skip to the main content before the assistive technology reads the full content of the interjection.
-
 We have added the role `list` and `listitem` to the `NavigationUl` and `NavigationList` respectively, due to a VoiceOver bug to reinstate the list semantics.
 
 We have also added visually hidden text to let the user know which item in the Navigation is the current page. Note the use of visually hidden text here is due to lack of support at this time for the aria-current page attribute. Also note the use of `role="text"` to stop splitting in VoiceOver.
 
+In the screen reader UX only the menu button and its content should be available to assistive technology. The exposed swipeable navigation should be hidden. To achieve this we add `aria-hidden:true` to the exposed swipeable navigation so that this is not expose to these users and also add `tabindex=-1` to the links contained within this to remove them from the tab order.
```

Comment:
Can we replace the word `exposed` with `visible`?

Reply:
I think `exposed` is more appropriate in this context because the navigation will be still visible, but not exposed to screen readers.

Reply:
Aaah ok. I was misinterpreting what was meant by the statement because of the use of `exposed` and `hidden` in the same sentence. Would it be better to restructure the sentence a little.

```suggestion
In the screen reader UX only the menu button and its content should be available to assistive technology. The swipeable navigation is exposed to assistive technologies and  should be hidden. To achieve this we add `aria-hidden=true` to the swipeable navigation. We also add `tabindex=-1` to the links contained within this to remove them from the tab order.
```

Reply:
I renamed it in a different way, hope this is more clear.